doc(release) Release notes web 24.04.12 (#4246)
centreon/centreon-documentation: versioned_docs/version-24.04/releases/centreon-os.mdx

## Centreon Web

### 24.04.12

Release date: `March 31, 2025`

<details open>
  <summary>Enhancements</summary>

- [Resource Status] Optimized search query.

</details>

<details>
  <summary>Bug fixes</summary>

> - [API] Fixed an issue on configuration generation if an host have been patched with API.

Reviewer: ```suggestion
- [API] Fixed an issue where the configuration couldn't be generated if a host had been patched using the API.
```

> - [Configuration] Fixed ACL check on the contact / user configuration page.

Reviewer: ```suggestion
- [Configuration] Fixed ACL checks on contact/user configuration page.
```

> - [Configuration] Fixed an issue on notification period field which was not saved in some cases.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where the notification period field was not saved in some cases.
```

- [Custom Views] Fixed an issue preventing non admin users from seeing host groups in the Host group Monitoring widget and service groups in the Service group Monitoring widget.

> - [Downtimes] Fixed an issue on displaying the downtimes pages for users under ACLs.

Reviewer: ```suggestion
- [Downtimes] Fixed an issue where non admin users could not display the downtimes page.
```

> - [Downtimes] Fixed an issue where loading downtimes page resulted in gateway timeout error.

Reviewer: ```suggestion
- [Downtimes] Fixed an issue where loading the downtimes page resulted in a gateway timeout error.
```

> - [Service groups] Fixed issue preventing non admin users from displaying Service Groups.

Reviewer: ```suggestion
- [Service groups] Fixed an issue preventing non admin users from displaying Service Groups.
```

> - [Top counter] Fixed an issue where calls were performed even if user hadn't access.

Reviewer: ```suggestion
- [Top counter] Unnecessary calls are not longer performed when users do not have access to the Top Counter.
```
